open_discussions_api/channels/client.py

```diff
@@ -39,3 +41,25 @@ def create(self, **channel_params):
             self.get_url("/channels/"),
             json=channel_params
         )
+
+    def add_contributor(self, channel_name, username):
+        """Add a contributor to a channel"""
```

Comment:
Could you add arg docs to these methods?

Reply:
yes